book/src/appendix/contributing/rust.md

```
@@ -5,12 +5,12 @@
 To start working with [this repository](https://github.com/vlayer-xyz/vlayer), you will need to install following software:
 
 - [Rust](https://www.rust-lang.org/tools/install) compiler
-- [Risc-0 toolchain](https://dev.risczero.com/api/zkvm/quickstart) with VM version v2.1.0 and Rust toolchain version v1.85.0
+- [Risc-0 toolchain](https://dev.risczero.com/api/zkvm/quickstart) with VM version v2.1.0 and Rust toolchain version v1.88.0
   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)